docs/running-locally.md

```diff
@@ -227,6 +227,14 @@ git commit --signoff -m 'feat: Added a new feature. Fixes #1234'
   have checked out your code into `$GOPATH/src/github.com/argoproj/argo-workflows`.
 * If you encounter "out of heap" issues when building UI through Docker, please validate resources allocated to Docker.
   Compilation may fail if allocated RAM is less than 4Gi.
+* To start profiling with [pprof](https://go.dev/blog/pprof), pass `ARGO_PPROF=true` when starting the controller locally,
+  and run the following:
```

Comment:
```suggestion
* To start profiling with [`pprof`](https://go.dev/blog/pprof), pass `ARGO_PPROF=true` when starting the controller locally.
  Then run the following:
```

I believe you won't need the `.spelling` change if it is in backticks. Also fix run-on sentence